pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
```

Comment:
what is the purpose of adding poetry here ? 

If it's for deploying new versions of the package i'm currently working on a github action to deploy from Github -> PyPI

Reply:
@zakhar-kogan do you need poetry for your local development ?

Reply:
> @zakhar-kogan do you need poetry for your local development ?

Not necessarily; I can manage without it + at least add `pyproject.toml` and `poetry.lock` to .gitignore. Should I do it?

Reply:
yes lets do that

Reply:
Sure, will do that in a separate commit/PR

Reply:
lets do this in another PR, and no need for a gitignore